modules/compliance-objects.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-operator-advanced.adoc
+
+[id="compliance-objects_{context}"]
+= Using the ComplianceSuite and ComplianceScan objects directly
+
+While it is recommended that users take advantage of the ScanSetting and ScanSettingBinding objects to define the Suites and Scans, there are valid use cases to define the ComplianceSuites directly:
+
+* Specifying only a single rule to scan. This can be useful for debugging together with the debug=true attribute which increases the OpenSCAP scanner verbosity, as the debug mode tends to get quite verbose otherwise. Limiting the test to one rule helps to lower the amount of debug information.
```

Comment:
should be used attribute in the following format `debug: true` to avoid confusion.